Makefile

```diff
@@ -6,7 +6,8 @@ ARCH = $(shell uname -p)
 
 ## dependencies for e2e-tests
 E2E_VOLUME_MOUNT     ?=
-E2E_BUILD_SPIN       ?= false
+E2E_USE_CANARY       ?= false ## Whether to use the canary build instead of the local build
+E2E_BUILD_SPIN 	     =  $(if $E2E_USE_CANARY == "false",true,false)
```

Comment:
wondering if we should use built binary in CI instead of building again. Downside is that on Github actions, the build jobs are really slow.